src/SearchResults.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import TableHeader from "./TableHeader";
+import TableData from "./TableData";
+import FakeBookings from "./data/fakeBookings.json";
+
+function SearchResults() {
+  return (
+    <div>
+      <table class="table table-striped">
+        <TableHeader />
+        <TableData results={FakeBookings} />
```

Comment:
Love the fact that you've separated your components out into different components. 😄 
I'd call `TableData` something like `Table` or `TableDataDisplay` as it represents the view of the data and _not_ strictly the data itself